bazel/BUILD.plasma

```diff
@@ -0,0 +1,82 @@
+load("@com_github_google_flatbuffers//:build_defs.bzl", "flatbuffer_cc_library")
+
+cc_library(
+    name = "plasma",
+    visibility = ["//visibility:public"],
+    hdrs = [
+        "cpp/src/plasma/client.h",
```

Comment:
What's the convention for when to list all the files versus to use `*.h`?

Reply:
no convention, if all the .h are included, I put *.h, but if it's only a small subset I list them